server/src/main/java/org/apache/uniffle/server/Checker.java

```diff
@@ -17,13 +17,6 @@
 
 package org.apache.uniffle.server;
 
-public abstract class Checker {
-
-  protected ShuffleServerConf conf;
-
-  Checker(ShuffleServerConf conf) {
-    this.conf = conf;
-  }
-
-  abstract boolean checkIsHealthy();
+public interface Checker {
```

Comment:
Rename to `StorageChecker` ? And this part could be moved into Storage's checker package? WDYT

Reply:
And why we need to change this class to interface? It looks fine for me.

Reply:
> Rename to `StorageChecker` ? And this part could be moved into Storage's checker package? WDYT

`HealthCheck` is also in `org.apache.uniffle.server`, should we move that also?

> And why we need to change this class to interface? It looks fine for me.

The protected field `conf` is never read. But seems the failed test is relying on constructor.